src/network/errors.ts

```diff
@@ -141,25 +54,8 @@ class ErrorDNSResolver<T> extends ErrorNetwork<T> {
 
 export {
   ErrorNetwork,
-  ErrorProxy,
-  ErrorProxyNotRunning,
-  ErrorProxyConnectInvalidUrl,
-  ErrorProxyConnectMissingNodeId,
-  ErrorProxyConnectAuth,
-  ErrorConnection,
-  ErrorConnectionNotRunning,
-  ErrorConnectionComposed,
-  ErrorConnectionNotComposed,
-  ErrorConnectionMessageParse,
-  ErrorConnectionTimeout,
-  ErrorConnectionEndTimeout,
-  ErrorConnectionNodesEmpty,
-  ErrorConnectionStart,
-  ErrorConnectionStartTimeout,
-  ErrorConnectionStartTimeoutMax,
-  ErrorConnectionCompose,
-  ErrorConnectionComposeTimeout,
   ErrorCertChain,
+  ErrorCertVerificationError,
```

Comment:
I added a check to verification  to check if at least 1 NodeId was provided for verification. None of the other errors applied. I can't use `ErrorCertChain` since it doesn't specify a code or description.

it could use a better name.

Reply:
Ok well then it should be still under `ErrorCertChain` hierarchy right? Just specify exactly what you are doing here, you say at least 1 node id, well you can said `ErrorCertChainNodeIdMissing` or something similar.